include/wpe/input.h

```diff
@@ -60,13 +60,22 @@ enum wpe_input_pointer_event_type {
     wpe_input_pointer_event_type_button,
 };
 
+enum wpe_input_pointer_modifier {
+    wpe_input_pointer_modifier_button1 = 1 << 0,
+    wpe_input_pointer_modifier_button2 = 1 << 1,
+    wpe_input_pointer_modifier_button3 = 1 << 2,
+    wpe_input_pointer_modifier_button4 = 1 << 3,
+    wpe_input_pointer_modifier_button5 = 1 << 4,
+};
```

Comment:
I suppose there will not ever be more than 5 buttons, because I suppose this maps to [MouseEvent.buttons](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons) (in the DOM). If that's the case, wouldn't it make more sense to call these `enum wpe_input_pointer_button`, `wpe_input_pointer_button1`, and so on?

Reply:
No, because you can use the enum to represent several buttons pressed at the same time, that's why they are flags. See the WebKit patch, this eum is used as a modifier.

Reply:
And it's consistent with wpe_input_keyboard_modifier

Reply:
It is a bitfield indicating which buttons are pressed, but they are not “modifiers” because they don't modify the meaning of the event in any way—they are just pressed or not. In my mind it sounds more accurate to remove the “modifiers” word from the name, but I won't block the merge on that.

Reply:
That's a good point.